en/06_Project_Governance/03_Maintainer_Guidelines.md

```diff
@@ -46,59 +26,37 @@ or any additional guidelines outlined via `README` and `CONTRIBUTING` files.
+The CMS Squad require write access to core repositories to maintain their pace,
+often working alongside Core Committers.
 
+CMS Squad members have write access to core repositories in order to work effectively with GitHub issues. They are expected to use those permissions with good judgement for merging pull requests.
 
 ## Guidelines
 
 With great power (write access) comes great responsibility.
 
 First and foremost rule of a maintainer is to collaborate with other maintainers. Follow the house rules and remember that others also care about the modules they maintain.
 
-
 ### House rules overview
 
- * Be friendly, encouraging and constructive towards other community members
- * Frequently review pull requests and new issues (in particular, respond quickly to @mentions)
- * Treat issues according to our [issue guidelines](issues_and_bugs), and use the [triage resources](triage_resources)
- * Don't commit directly to a release branch, raise pull requests instead (except trivial fixes)
- * Only merge code you have tested and fully understand. If in doubt, ask for a second opinion.
- * Follow the [Supported Modules Standard](https://www.silverstripe.org/software/addons/supported-modules-definition/)
- * Ensure contributions have appropriate [test coverage](../developer_guides/testing), are documented, and pass our [coding conventions](/getting_started/coding_conventions)
- * Keep the codebase "releasable" at all times (check our [release process](release_process))
- * Follow [Semantic Versioning](code/#picking-the-right-version) by putting any changes into the correct branch
- * API changes and non-trivial features should not be merged into release branches. 
- * API changes on master should not be merged until they have the buy-in of at least two Core Committers (or better, through the [core mailing list](https://groups.google.com/forum/#!forum/silverstripe-dev))
- * Be inclusive. Ensure a wide range of Silverstripe CMS developers can obtain an understanding of your code and docs, and you're not the only one who can maintain it.
- * Avoid `git push --force`, and be careful with your git remotes (no accidental pushes)
- * Use your own forks to create feature branches
- * We release using the standard process. See the [Making a Silverstripe CMS Core Release](making_a_silverstripe_core_release)
-
+* Be friendly, encouraging and constructive towards other community members
+* Frequently review pull requests and new issues (in particular, respond quickly to @mentions)
+* Treat issues according to our [issue guidelines](/contributing/issues_and_bugs/), and use the [triage resources](/contributing/triage_resources/)
+* Don't commit directly to a release branch, raise pull requests instead
+* Only merge code you have tested and fully understand. If in doubt, ask for a second opinion.
+* Follow the [Supported Modules Standard](https://www.silverstripe.org/software/addons/supported-modules-definition/)
```

Comment:
> Follow the [Supported Modules Standard](https://www.silverstripe.org/software/addons/supported-modules-definition/)

We don't _really_ follow that, most likely - but as long as that page exists, I don't think we should stop referring to it. We should instead holistically reconsider how we approach supported modules.